src/test/php/PHPMD/Renderer/SARIFRendererTest.php

```diff
@@ -69,13 +70,19 @@ public function testRendererCreatesExpectedNumberOfJsonElements()
         $renderer->renderReport($report);
         $renderer->end();
 
+        self::assertJson(
+            $writer->getData(),
+            'The writer json is invalid: ' . $writer->getData()
+        );
+        $sarifContent = file_get_contents(__DIR__.'/../../../resources/files/renderer/sarif_renderer_expected.sarif');
+        self::assertJson(
+            $sarifContent,
+            'The sarif json is invalid: ' . $sarifContent
+        );
+
         $this->assertJsonEquals(
             $writer->getData(),
-            'renderer/sarif_renderer_expected.sarif',
-            function ($actual) {
-                $actual['runs'][0]['tool']['driver']['version'] = '@package_version@';
-                return $actual;
-            }
+            'renderer/sarif_renderer_expected.sarif'
```

Comment:
I guess this should be reverted to pass the test:
```suggestion
            'renderer/sarif_renderer_expected.sarif',
            function ($actual) {
                $actual['runs'][0]['tool']['driver']['version'] = '@package_version@';
                return $actual;
            }
```